### DISCUSSION TITLE
v1.0.218 BB apps slow with timeouts

### DISCUSSION THREAD

**Discussion by mielune** [Budibase/budibase#6598] [Help]
Hi folks,

Since upgraded to 1.0.218, queries to databases (internal or postgres) are slow with progress circles on each data provider.
Lot of "Query response time exceeded 10000ms timeout."

It is linked to requests like 
`POST https://my.bb/api/v2/queries/query_datasource_plus_acc14600a87a4c3ba0fb83fa9c9a7a50_916b4b7672ed4490aface72fbf0020df`
Status is 400

No special logs from BB services.

Thank you

**Comment by NicolasFabre81** [comment]
@melohagan Couldn't this issue be related to https://github.com/Budibase/budibase/issues/6580 ?

**Reply by melohagan** [reply]
Thanks for sharing! It is possible.
